custom/ilsgateway/tests/test_migration_task.py

```diff
@@ -36,12 +38,48 @@ def setUp(self):
     def tearDown(self):
```

Comment:
No, tearDownClass also exists in this class: https://github.com/dimagi/commcare-hq/pull/9803/files#diff-33340a512152179cc8b9dddba5cecc38R83